PROCESS TOOK 458 SECONDS SUCCESS in  benchmark_MACE.py  AND DSNAME  14
PROCESS TOOK 458 SECONDS SUCCESS in  benchmark_MACE.py  AND DSNAME  14
